File: app/src/main/java/com/cr4sh/nhlauncher/ColorPickerMod.java
package com.cr4sh.nhlauncher;

import android.annotation.SuppressLint;
import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;

import com.skydoves.colorpickerview.ColorPickerView;

public class ColorPickerMod extends ColorPickerView {
    public ColorPickerMod(Context context) {
        super(context);
    }

    public ColorPickerMod(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public ColorPickerMod(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    // Override touch event of ColorPickerView so we can use it inside ViewPager
    @SuppressLint("ClickableViewAccessibility")
    @Override
    public boolean onTouchEvent(MotionEvent event) {
        getParent().requestDisallowInterceptTouchEvent(true);
        return super.onTouchEvent(event);
    }

}

File: app/src/main/java/com/cr4sh/nhlauncher/overrides/ColorPickerMod.kt
package com.cr4sh.nhlauncher.overrides

import android.annotation.SuppressLint
import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import com.flask.colorpicker.ColorPickerView

class ColorPickerMod : ColorPickerView {
    constructor(context: Context?) : super(context)
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    )

    // Override touch event of ColorPickerView so we can use it inside ViewPager
    @SuppressLint("ClickableViewAccessibility")
    override fun onTouchEvent(event: MotionEvent): Boolean {
        parent.requestDisallowInterceptTouchEvent(true)
        return super.onTouchEvent(event)
    }
}
